requirements.txt

```diff
@@ -0,0 +1,13 @@
+bumpversion==0.5.3
+coverage==4.4.2
+pre-commit>=2.12.0,<2.13
+mock==2.0.0
```

Comment:
This is in the py3 stdlib now. No need to install it on its own.

Reply:
removed